### PR TITLE
Updated HTTPServerResponse docs

### PR DESCRIPTION
**Pull request by KyeMaloy97**
Updated the docs by adding code examples to the HTTPServerResponse class

### CODE REVIEW

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift [hunk 1]
```diff
@@ -18,8 +18,27 @@ import Foundation
...
 // MARK: HTTPServerResponse
 
-/// This class implements the `ServerResponse` protocol for outgoing server
-/// responses via the HTTP protocol.
+/**
+This class implements the `ServerResponse` protocol for outgoing server
+responses via the HTTP protocol. Data and Strings can be written.
+
+### Usage Example: ###
+````swift
```
Comment (helenmasters) [inline Sources/KituraNet/HTTP/HTTPServerResponse.swift]: We should have an explanation here of what the example is doing, even if it's only brief.

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift [hunk 2]
```diff
@@ -92,18 +139,32 @@ public class HTTPServerResponse : ServerResponse {
         }
     }
 
-    /// Write a string and end sending the response.
-    ///
-    /// - Parameter text: String to write to a socket.
-    /// - Throws: Socket.error if an error occurred while writing to a socket.
+    /**
+    Write a string and end sending the response.
```
Comment (helenmasters) [inline Sources/KituraNet/HTTP/HTTPServerResponse.swift]: Write a string and complete sending the response.

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift [hunk 3]
```diff
@@ -92,18 +139,32 @@ public class HTTPServerResponse : ServerResponse {
...
     public func end(text: String) throws {
         try write(from: text)
         try end()
     }
     
-    /// End sending the response.
-    ///
-    /// - Throws: Socket.error if an error occurred while writing to a socket.
+    /**
+    End sending the response.
```
Comment (helenmasters) [inline Sources/KituraNet/HTTP/HTTPServerResponse.swift]: "Complete sending the response." I think is clearer than "End sending the response."

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift [hunk 4]
```diff
@@ -205,7 +266,14 @@ public class HTTPServerResponse : ServerResponse {
         }
     }
     
-    /// Reset this response object back to its initial state
+    /**
+    Reset this response object back to its initial state
```
Comment (helenmasters) [inline Sources/KituraNet/HTTP/HTTPServerResponse.swift]: Please can you put a full stop on the end of this sentence.

Comment (KyeMaloy97) [reply Sources/KituraNet/HTTP/HTTPServerResponse.swift]: ✅

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift (hunk 3 above)
Comment (KyeMaloy97) [reply Sources/KituraNet/HTTP/HTTPServerResponse.swift]: ✅

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift (hunk 2 above)
Comment (KyeMaloy97) [reply Sources/KituraNet/HTTP/HTTPServerResponse.swift]: ✅

File: Sources/KituraNet/HTTP/HTTPServerResponse.swift (hunk 1 above)
Comment (KyeMaloy97) [reply Sources/KituraNet/HTTP/HTTPServerResponse.swift]: ✅

**Pull request merged**